localstack/services/events/provider_v2.py

```diff
@@ -58,18 +319,358 @@ def put_rule(
+            if end_index < input_dict_len
+            else None
+        )
+        return limited_dict, next_token
+
+    def _extract_event_bus_name(
+        self, event_bus_name_or_arn: EventBusNameOrArn | None
+    ) -> EventBusName:
+        """Return the event bus name. Input can be either an event bus name or ARN."""
+        if not event_bus_name_or_arn:
+            return "default"
+        return event_bus_name_or_arn.split("/")[-1]
+
+    def _event_bust_dict_to_api_type_list(self, event_buses: EventBusDict) -> EventBusList:
+        """Return a converted dict of EventBus model objects as a list of event buses in API type EventBus format."""
+        event_bus_list = [
+            self._event_bus_dict_to_api_type_event_bus(event_bus)
+            for event_bus in event_buses.values()
+        ]
+        return event_bus_list
+
+    def _event_bus_dict_to_api_type_event_bus(self, event_bus: EventBus) -> ApiTypeEventBus:
+        if event_bus.policy:
+            event_bus = {
+                "Name": event_bus.name,
+                "Arn": event_bus.arn,
+                "Policy": event_bus.policy,
+            }
+        else:
+            event_bus = {
+                "Name": event_bus.name,
+                "Arn": event_bus.arn,
+            }
+        return event_bus
+
+    def _delete_rule_workers(self, rules: RuleDict | Rule) -> None:
+        if isinstance(rules, Rule):
+            rules = {rules.name: rules}
+        for rule in rules.values():
+            self._rule_workers.pop(rule.arn)
+
+    def _rule_dict_to_api_type_list(self, rules: RuleDict) -> RuleResponseList:
+        """Return a converted dict of Rule model objects as a list of rules in API type Rule format."""
+        rule_list = [self._rule_dict_to_api_type_rule(rule) for rule in rules.values()]
+        return rule_list
+
+    def _rule_dict_to_api_type_rule(self, rule: Rule) -> ApiTypeRule:
+        rule = {
+            "Name": rule.name,
+            "Arn": rule.arn,
+            "EventPattern": rule.event_pattern,
+            "State": rule.state,
+            "Description": rule.description,
+            "ScheduleExpression": rule.schedule_expression,
+            "RoleArn": rule.role_arn,
+            "ManagedBy": rule.managed_by,
+            "EventBusName": rule.event_bus_name,
+        }
+        return {k: v for k, v in rule.items() if v is not None and v != {} and v != []}
+
+    def _delete_target_workers(self, ids: TargetIdList | TargetId, rule) -> None:
+        if isinstance(ids, TargetId):
+            ids = [ids]
+        for target_id in ids:
+            if target := rule.targets.get(target_id):
+                target_arn = target["Arn"]
+                try:
+                    del self._target_workers[target_arn]
+                except KeyError:
+                    LOG.error(f"Error deleting target worker {target_arn}.")
+
+    def _put_entries(
```

Comment:
Rename to a more meaningful function name, also include docstring

Reply:
maybe dispatch_events, deliver_events, ...?

The API is called put_entries but that's from a user's perspective. Within the EventBridge service, using active terminology such as dispatch makes more sense. Just reading over the AWS docs and drafting a docstring might help decide upon a better name.

Reply:
renamed in: https://github.com/localstack/localstack/pull/10664/commits/da0f12a0405588b61d29283965634af5c3d8a66b